src/components/AdminNavbar/AdminNavbar.test.tsx

```diff
@@ -189,48 +189,4 @@ describe('Testing Admin Navbar', () => {
 
     await wait();
   });
-
-  // eslint-disable-next-line jest/no-commented-out-tests
-  // test('Should check if organisation image is present', async () => {
-  //   const { container } = render(
-  //     <MockedProvider addTypename={false} link={link2}>
-  //       <BrowserRouter>
-  //         <Provider store={store}>
-  //           <I18nextProvider i18n={i18nForTest}>
-  //             <AdminNavbar {...props} />
-  //           </I18nextProvider>
-  //         </Provider>
-  //       </BrowserRouter>
-  //     </MockedProvider>
-  //   );
-
-  //   expect(container.textContent).not.toBe('Loading data...');
-  //   await wait();
-
-  //   const imageOptions = screen.getByTestId(/navbarOrgImagePresent/i);
-  //   const imageLogo = screen.getByTestId(/orgLogoPresent/i);
-  //   expect(imageLogo).toBeInTheDocument();
-  //   expect(imageOptions).toBeInTheDocument();
-  // });
-
-  test('Should check if organisation image is not present', async () => {
-    const { container } = render(
-      <MockedProvider addTypename={false} link={link1}>
```

Comment:
We should be checking for the presence of the new image.

Reply:
We have changed the image to a static navbar hamburger icon so there was no need to check the presence because it will always be there and  the functionality of the icon is limited to only toggling a menu. As such, there may be little value in writing automated tests specifically for this icon.

Alternatively I could wrap a stand alone icon in a span like I did with "More" and bring back the test, I just didn't think it was necessary.
please let me know what you think.